src/consts/general.consts.ts

```diff
@@ -0,0 +1,48 @@
+import { Settings } from "react-slick";
+
+export const CAROUSEL_SETTINGS: Settings = {
```

Comment:
Dit is een voorbeeld van settings die ge aan de slider meegeeft. Geen idee wat dat dit allemaal is niemeer, te lang geleden dak da gemaakt heb hahaha